### PR TITLE
Add configuration to run georchestra with multiple jetty instances

### PR DESCRIPTION
**Pull request by Vampouille**
Currently, jetty was configured for following modules : 

* Security proxy
* Cas
* mapfishapp
* extractorapp
* header

### CODE REVIEW

File: extractorapp/pom.xml [hunk 1]
```diff
@@ -356,16 +356,15 @@
...
       <plugin>
-        <groupId>org.mortbay.jetty</groupId>
-        <artifactId>maven-jetty-plugin</artifactId>
-        <version>6.1.26</version>
+        <groupId>org.eclipse.jetty</groupId>
+        <artifactId>jetty-maven-plugin</artifactId>
+        <version>9.2.11.v20150529</version>
         <configuration>
-          <webApp>${basedir}/target/extractorapp.war</webApp>
-          <scanIntervalSeconds>5</scanIntervalSeconds>
```
Comment (fvanderbiest) [inline extractorapp/pom.xml]: scanIntervalSeconds 5 was removed ?

File: extractorapp/pom.xml [hunk 2]
```diff
@@ -356,16 +356,15 @@
...
-          <scanTargets>
-            <scanTarget>src/main/java</scanTarget>
-            <scanTarget>src/main/resources</scanTarget>
-          </scanTargets>
```
Comment (fvanderbiest) [inline extractorapp/pom.xml]: scanTargets removed too ?

**Comment by fvanderbiest** [comment]
LGTM otherwise.
Thanks !

**Comment by Vampouille** [comment]
It works well in my ide without scan interval.

**Comment by Vampouille** [comment]
This new commit add and/or unify scan interval on almost all modules

**Comment by fvanderbiest** [comment]
Thanks Julien, looks good to me now !

**Pull request merged**